lib/tasks/taxonomy/find_nested_tags.rake

```diff
@@ -0,0 +1,55 @@
+require 'gds_api/rummager'
+require 'gds_api/content_store'
+require 'set'
+
+namespace :taxonomy do
+  desc "Find any content tagged at one taxon level, then tagged again at one of those taxon's ancestors"
+  task :find_nested_tags, [:root_taxon_base_path] => [:environment] do |_, args|
+    root_taxon_base_path = args[:root_taxon_base_path]
+
+    content_store = GdsApi::ContentStore.new(Plek.find('content-store'))
+
+    root_taxon = content_store.content_item(root_taxon_base_path).to_h
```

Comment:
You should probably get it from the publishing API to avoid another dependency, as this is a publishing app. That means either passing in a content_id, or do a search by base_path.

An example with content ID would be:

```
Services.publishing_api.get_content(taxon_content_id).to_h
```